mmv1/third_party/terraform/resources/resource_dataflow_job.go.erb

```diff
@@ -366,8 +366,21 @@ func resourceDataflowJobRead(d *schema.ResourceData, meta interface{}) error {
 	if err := d.Set("service_account_email", optionsMap["serviceAccountEmail"]); err != nil {
 		return fmt.Errorf("Error setting service_account_email: %s", err)
 	}
-	if err := d.Set("additional_experiments", optionsMap["experiments"]); err != nil {
-		return fmt.Errorf("Error setting additional_experiments: %s", err)
+	if addExper, ok := d.GetOk("additional_experiments"); ok {
```

Comment:
I'd like to suggest removing this section altogether and just not setting this field on read at all.